demos/appyx-interactions/android/src/main/kotlin/com/bumble/appyx/interactions/MainActivity.kt

```diff
@@ -25,7 +25,7 @@ import androidx.compose.ui.unit.ExperimentalUnitApi
 import androidx.compose.ui.unit.dp
 import com.bumble.appyx.components.demos.cards.DatingCards
 import com.bumble.appyx.components.demos.promoter.PromoterExperiment
-import com.bumble.appyx.components.internal.testdrive.TestDriveExperiment
+import com.bumble.appyx.components.internal.testdrive.android.TestDriveExperiment
```

Comment:
Does it need to be android specific?

Reply:
We can use the generic one instead but we'll need to provide the required arguments. But I preferred to add that intermediate layer and keep the `when` statement where `TestDriveExperiment` is used readable. Otherwise it would turn to something like this:
```
  when (content) {
      0 -> DatingCards()
      1 -> SpotlightExperiment()
      2 -> SpotlightExperimentDebug()
      3 -> BackStackExperimentDebug()
      4 -> TestDriveExperiment(
        screenWidthPx = (LocalConfiguration.current.screenWidthDp * LocalDensity.current.density).roundToInt(),
        screenHeightPx = (LocalConfiguration.current.screenHeightDp * LocalDensity.current.density).roundToInt(),
        colors = colors,
        element = InteractionTarget.Child1,
        modifier = Modifier.fillMaxSize().background(appyx_dark),
      )
      5 -> PromoterExperiment()
      else -> SpotlightExperiment()
  }
```